source/_patterns/02-molecules/vue-widget/src/vuex-cryptos/index.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-param-reassign, no-shadow */
```

Comment:
Would we normally have to disable these ESLint rules for Vuex mutations because of how they mutate the state?